docker.go

```diff
@@ -113,7 +113,7 @@ func (docker *dockerConfig) call() int {
 			"-e", fmt.Sprintf("HOME=%v", mountingHome),
 		}...)
 	} else {
-		dockerArgs = append(dockerArgs, "-e", fmt.Sprintf("HOME=%s/home/%s", dockerMountImagePath, currentUser.Username))
+		// dockerArgs = append(dockerArgs, "-e", fmt.Sprintf("HOME=%s/home/%s", dockerMountImagePath, currentUser.Username))
```

Comment:
Why not remove the `else`?
If we don't remove it, this needs a comment

Reply:
Done: f5b1f7b